zappa/core.py

```diff
@@ -318,13 +318,17 @@ def __init__(
             self.manylinux_suffix_start = "cp310"
 
         # AWS Lambda supports manylinux1/2010, manylinux2014, and manylinux_2_24
-        manylinux_suffixes = ("_2_24", "2014", "2010", "1")
+        # Currently python3.7 lambda runtime does not support manylinux_2_24
+        # See https://github.com/zappa/Zappa/issues/1249 for more details
+        if self.runtime == "python3.7":
```

Comment:
Can you add a testcase for this, where python3.7 and `_2_24` wheel is NOT matched?